release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -113,6 +113,16 @@ For more information, see xref:../installing/installing_bare_metal_ipi/ipi-insta
 
 For more information, see xref:../installing/installing_bare_metal_ipi/ipi-install-installation-workflow.adoc#configuring-the-bios_ipi-install-installation-workflow[Configuring the BIOS] and xref:../installing/installing_bare_metal_ipi/ipi-install-installation-workflow.adoc#configuring-the-raid_ipi-install-installation-workflow[Configuring the RAID].
 
+[id="ocp-4-11-installation-vsphere-csi"]
+==== CSI driver installation on vSphere clusters
+To install a CSI driver on a cluster running on vSphere, you must have the following components installed:
+
+* Virtual hardware version 15 or later
+* vSphere version 6.7 Update 3 or later
+* VMware ESXi version 6.7 Update 3 or later
+
+Components with versions earlier than those above are deprecated. These versions are still fully supported, but Red Hat recommends that you use vSphere 7.0 Update 2 or later and ESXi 7.0 Update 2 or later.
+
```

Comment:
@bscott-rh same update should also need in 4.11 vsphere/VMC[ installation doc](https://bscott-rh.github.io/openshift-docs/RN-411-esxi-prereqs/installing/installing_vsphere/installing-vsphere-installer-provisioned-customizations.html#installation-vsphere-infrastructure_installing-vsphere-installer-provisioned-customizations), how could we track that?

Reply:
@bscott-rh Hi, the situation is a little more complex that in OCP4.11:
Virtual hardware version earlier than 15 is **removed**,
vSphere version earlier than 6.7 Update 3 and VMware ESXi version earlier than 6.7 Update 3 are **removed**, 
vSphere version earlier than 7.0.2 and VMware ESXi version earlier than 7.0.2 are **deprecated**, 
(see "Deprecated and removed features" section in storage PR https://github.com/openshift/openshift-docs/pull/46711)

I suggest we like to mention as below, but feel free to update/reword:
==== CSI driver installation on vSphere clusters
To install a CSI driver on a cluster running on vSphere, you must have the following components installed:

* Virtual hardware version 15 or later
* vSphere version 7.0.2 or later
* VMware ESXi version 7.0.2 or later

Components with versions earlier than those above are deprecated or removed(Referring to the "Deprecated and removed features" section for detailed info). These deprecated versions are still fully supported, but Red Hat recommends that you use vSphere 7.0 Update 2 or later and ESXi 7.0 Update 2 or later. 

cc @gnufied

Reply:
Thank you for the detailed clarification. I have updated the RN with this feedback. PTAL

Reply:
Looks good to me, and thanks for the update.